.eslintrc.json

```diff
@@ -3,18 +3,20 @@
     "parser": "@typescript-eslint/parser",
     "plugins": [
         "@typescript-eslint",
-        "react-hooks"
+        "react-hooks",
+        "iot-cardboard-js"
     ],
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
         "prettier",
-        "prettier/@typescript-eslint"
+        "plugin:iot-cardboard-js/recommended"
     ],
     "rules": {
+        "iot-cardboard-js/debug-logging-disabled": "warn",
         "no-debugger": "warn",
-        "no-multiple-empty-lines": "error",
-        "react-hooks/exhaustive-deps": "off",
+        "no-multiple-empty-lines": "warn",
```

Comment:
just to confirm, are we sure about turning those off?

Reply:
I'm not actually turning anything off. I'm just setting them to warnings so that it won't fail your local compilation. WHat we see today is people adding the `eslint-ignore` statements just to be able to develop and then those can accidentally get checked in. This allows us to get away with it locally (with yellow squiggles) and then it will throw errors on the PR if you forget to remove them.